src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,119 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 1 << 4;
+    private static final int MAX_CAPACITY = 1 << 30;
```

Comment:
why do we need to process these operations before setting value to constant?

Reply:
It just a pointer to that the capacity must to be a power of two rather than any arbitrary value. Such capacity allows us to use a bitwise & operation to select the bucket index instead of a modulo %. The bitwise operators works faster because operate with bits directly.